spark/src/main/scala/geotrellis/spark/ingest/Reproject.scala

```diff
@@ -37,8 +37,8 @@ object Reproject {
         ((key, newExtent), newTile)
       }
 
-    val metaData = 
-      RasterMetaData.fromRdd(reprojectedTiles, destCRS, rdd.metaData.tileLayout) { key => key._2 }
+    val metadata =
```

Comment:
No, should be separate PR, but we should maintain consistency until we make the switch.